core/domain/android_validation_constants_test.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Any specific reason for placing the `android_validation_constants.py` file inside the domain layer?

Reply:
Not really, just that all the files that use this are also in the domain layer, so it seemed appropriate.